src/router/projects.js

```diff
@@ -587,4 +587,55 @@ export const MY_PROJECT_VIEWS = [
       },
     ],
   },
+  {
+    path: ':id/institutions/',
+    name: 'MyProjectInstitutionsView',
+    meta: {
+      itemName: 'institutions',
+      breadcrumbs: [
+        {
+          label: 'institutions',
+          to: 'MyProjectInstitutionsView',
+        },
+      ],
+    },
+    component: () =>
+      import(
+        /* webpackChunkName: "unpublished" */ '@/views/project/institutions/ListView.vue'
+      ),
+  },
+  {
```

Comment:
Let's rename this to `create` so that the path is `:id/institutions/create` and name is `MyProjectInstitutionsCreateView`